.github/workflows/ci.yml

```diff
@@ -62,21 +67,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: 'Install Node.js'
-        uses: actions/setup-node@v3
+      - name: Install 🔧
+        uses: ./.github/actions/setup
         with:
-          node-version: 18
-      - name: Download npm artifacts
+          node-version: 16
```

Comment:
Node 16 is EOL'ing in a few months; was this change intentional?

Reply:
Nope - copy pasta